lib/modules/userTagger.js

```diff
@@ -506,14 +507,11 @@ modules['userTagger'] = {
 					taggedUsers.reverse();
 				}
 				break;
+			case 'username':
+				modules['userTagger'].sortUsername(taggedUsers, this.descending);
```

Comment:
Oh, sorry, JSHint is expecting this:

    case 'username':
        /* falls through */
    default: